### ISSUE TITLE
[Feature, P2, #Wearables] Include minter address or CV username in wearable metadata.

### ISSUE THREAD

**Issue by conlan** [cryptovoxels/issues#61]
Not so much a problem not but will be as wearable copycats crop up.

When an item is minted, ideally CV stores the minter address or CV username along with it. This allows users to verify the authenticity of an item and ensure that it came from the original crafter.

**Comment by bnolan** [comment]
Great idea.

**Comment by bnolan** [comment]
```
{
  "name": "GoldenSword",
  "image": "http://cryptovoxels.local:9000/w/abd4effa1479f0a8f0072da0f22928e9fb1bad42/png",
  "description": "One of the first swords minted for cryptovoxels!",
  "attributes": {
    "vox": "http://cryptovoxels.local:9000/w/abd4effa1479f0a8f0072da0f22928e9fb1bad42/vox",
    "author": "Bullauge",
    "issues": 8,
    "rarity": "legendary"
  },
  "external_url": "https://www.cryptovoxels.com/wearables/1",
  "background_color": "FAD961"
}
```

Rarity is calculated like this:

```ts

  get rarity () {
    if (this.issues < 10) {
      return 'legendary'
    } else if (this.issues < 100) {
      return 'epic'
    } else if (this.issues < 1000) {
      return 'rare'
    } else {
      return 'common'
    }
  }
```

**Comment by bnolan** [comment]
Fixed in `2.30.3`!

**Comment by conlan** [comment]
Yes!
